scripts/mkcloud

```diff
@@ -1084,8 +1084,6 @@ Optional
     scenario_dir='/tmp' (default=${SCRIPTS_DIR}/scenarios/cloud\$(getcloudver)/)
         Full path to directory containing scenario file.
 EOUSAGE
-    # Source qa_crowbarsetup.sh, which is where onadmin_help lives
-    source ${SCRIPTS_DIR}/qa_crowbarsetup.sh
```

Comment:
It won't break help, it will print mkcloud help and then print no file or directory for the function call immediately after it.